README.md

```diff
@@ -173,19 +178,264 @@ The explanation functions accept both models and pipelines as input. If a model
 * LIME Explainer: LIME Explainer uses the state-of-the-art Local interpretable model-agnostic explanations [(LIME)](https://github.com/marcotcr/lime) algorithm to create local surrogate models. Unlike the global surrogate models, LIME focuses on training local surrogate models to explain individual predictions.
 
-Tabular Explainer: Used with tabular datasets, it currently employs the following logic to invoke the direct SHAP explainers:
+* Tabular Explainer: Used with tabular datasets, it currently employs the following logic to invoke the direct SHAP explainers:
 
 
-| Original Model   | Invoked Explainer  |
-|-----|-----|
-| Tree-based models | SHAP TreeExplainer|
-| Deep Neural Network models| SHAP DeepExplainer|
-| Linear models | SHAP LinearExplainer |
-| None of the above  | SHAP KernelExplainer |
+  | Original Model   | Invoked Explainer  |
+  |-----|-----|
+  | Tree-based models | SHAP TreeExplainer|
+  | Deep Neural Network models| SHAP DeepExplainer|
+  | Linear models | SHAP LinearExplainer |
+  | None of the above  | SHAP KernelExplainer |
 
 <a name=Example></a>
 
-# Example
+# Use Interpret-Community
+
+
+## Interpretability in training
+
+
+1. Train your model in a local Jupyter notebook.
+
+    ```python
+    # load breast cancer dataset, a well-known small dataset that comes with scikit-learn
+    from sklearn.datasets import load_breast_cancer
+    from sklearn import svm
+    from sklearn.model_selection import train_test_split
+    breast_cancer_data = load_breast_cancer()
+    classes = breast_cancer_data.target_names.tolist()
+    
+    # split data into train and test
+    from sklearn.model_selection import train_test_split
+    x_train, x_test, y_train, y_test = train_test_split(breast_cancer_data.data,            
+                                                        breast_cancer_data.target,  
+                                                        test_size=0.2,
+                                                        random_state=0)
+    clf = svm.SVC(gamma=0.001, C=100., probability=True)
+    model = clf.fit(x_train, y_train)
+    ```
+
+2. Call the explainer locally: To initialize an explainer object, you need to pass your model and some training data to the explainer's constructor. You can also optionally pass in feature names and output class names (if doing classification) which will be used to make your explanations and visualizations more informative. Here is how to instantiate an explainer object using `TabularExplainer`, `MimicExplainer`, and `PFIExplainer` locally. `TabularExplainer` is calling one of the four SHAP explainers underneath (`TreeExplainer`, `DeepExplainer`, `LinearExplainer`, or `KernelExplainer`), and is automatically selecting the most appropriate one for your use case. You can however, call each of its four underlying explainers directly.
```

Comment:
"`KernelExplainer`), and is automatically selecting" -> "`KernelExplainer`) and automatically selects"